main.go

```diff
@@ -0,0 +1,36 @@
+package main
```

Comment:
I am guessing this was added by accident. If its something you want to keep, consider adding it to the cmd directory

Reply:
Yes, I removed it.